### ISSUE TITLE
Windows support

### ISSUE THREAD

**Issue by richfitz** [richfitz/remoji#3]
I'm getting test failures on windows 7; emoji conversion is not working there.
![screenshot_2015-10-29_16-33-52](https://cloud.githubusercontent.com/assets/1558093/10825005/f86e7d6a-7e5a-11e5-877a-a58e6466f4d5.png)

Happens in both Rstudio and Rgui (under VMware as I don't have a native machine).  Similar failure happened on winbuilder.

**Comment by gaborcsardi** [comment]
I'll help testing next week, I have windows at work.
It might just not work on windows, especially not in the cmd terminal.

**Comment by yixuan** [comment]
It seems to be related to the Unicode support on Windows. Actually `remoji` generated the correct byte sequence `\xf0\x9f\x91\8d`, but the console cannot recognize it.

**Comment by gaborcsardi** [comment]
It seems that windows is a mess and mostly unsupported, and we would need a workaround.

E.g. using the emoji name between colons, or an empty string, when an emoji is outputted. Or we can just use ASCII replacements for some emojis at least. E.g. smileys, and the rest can be empty string.

**Comment by richfitz** [comment]
Yeah, sounds good.  I'll have another look at windows this week and sort things out a little better.  After a couple of weeks dealing with more serious packages I could probably afford to do this package a bit in my train work.

**Comment by richfitz** [comment]
I'll see if [this update](https://support.microsoft.com/en-gb/kb/2729094) helps at all.

**Comment by gaborcsardi** [comment]
Hmm, interesting. If this fixes the fonts, can we detect whether it is
installed?

I am afraid that the encoding conversion code in R will still fail, at
least for most emojis. Anyway, we'll see....

**Comment by richfitz** [comment]
No effect here; still just a mess.  I wonder if @kevinushey knows if it's possible at all?  Or are we at the mercy of what Windows is willing to render here?

**Comment by kevinushey** [comment]
Unfortunately I don't know a great solution here, UTF-8 on Windows (and in R + RStudio) is kind of a mess. :/

Surprisingly, R complains if you try to actually construct a string literal using that representation:

``` R
> "\xf0\u009f\u0091\u008d"
Error: mixing Unicode and octal/hex escapes in a string is not allowed
```

Just in case that's relevant...
